docs/getting-started.rst

```diff
@@ -1,4 +1,11 @@
 Getting Started
 ===============
 
+.. no CERN physicist? sorry, won't work
```

Comment:
.. no CERN account? sorry, won't work - if you're interested in working with CERN data you are welcome to check out CODP
.. interested in joining us or building your own preservation service? contact us or check out our GitHub page

something like that? (it's just comments for now)